sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/SearchClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+
+public class SearchClientJavaDocCodeSnippets {
+    private SearchClient searchClient = new SearchClientBuilder().buildClient();
+
+    /**
+     * Code snippet for creating a {@link SearchClient}.
+     */
+    public void createSearchClientFromBuilder() {
+        // BEGIN: com.azure.search.documents.SearchClient.instantiation
+        SearchClient searchClient = new SearchClientBuilder()
+            .credential(new AzureKeyCredential("{key}"))
+            .endpoint("{endpoint}")
+            .buildClient();
+        // END: com.azure.search.documents.SearchClient.instantiation
+    }
+
+    /**
+     * Code snippet for creating a {@link SearchClient} with pipeline.
+     */
+    public void createSearchClientWithPipeline() {
+        // BEGIN: com.azure.search.documents.SearchClient.pipeline.instantiation
+        HttpPipeline pipeline = new HttpPipelineBuilder()
+            .policies(/* add policies */)
```

Comment:
Should we show this adding the credential policy? When using an `HttpPipeline` in the builder we don't add the credential information into the pipeline.